sdcm/nemesis.py

```diff
@@ -1858,6 +1853,31 @@ def disrupt_corrupt_then_scrub(self):
         self.target_node.stop_scylla_server(verify_up=False, verify_down=True)
         self.target_node.start_scylla_server(verify_up=True, verify_down=False)
 
+    def disrupt_grow_shrink_cluster(self):
+        add_nodes_number = self.tester.params.get('add_node_cnt')
```

Comment:
reusing this parameter might be problematic, I think we need a new one 
since it's logic so far is how many node to add after initial creation of the cluster.

for example the 5000 table test is setting this as 5.
and recently @roydahan has enabled nemesis on it.
so each time this nemesis would run it would create 5 new nodes (like the 5000 tables doesn't have enough issues on it's own)